src/Rokt-Kit.js

```diff
@@ -155,6 +175,25 @@ var constructor = function () {
         self.launcher.selectPlacements(selectPlacementsOptions);
     }
 
+    /**
+     * Sets extension data for Rokt Web SDK
+     * @param {Object} partnerExtensionData - The extension data object containing:
+     * - [extensionName] {string}: Name of the extension
+     * - [extensionName].options {Object}: Key-value pairs of options for the extension
+     * @returns {void} Nothing is returned
+     */
+    function setExtensionData(partnerExtensionData) {
+        if (!isInitialized()) {
+            console.error('Rokt Kit: Not initialized');
+            return;
+        }
+
+        // TODO: Should we check if select placements has been called?
+        // Some extensions seem to need that to happen first
+        // TODO: Should we attach the Rokt SDK to the kit as well?
```

Comment:
not if there isn't a reason to.  the paradigm is currently core calls kit. kit calls SDK.  I think we should keep it that way.